.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,55 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.177.0/containers/java
+{
+	"name": "Trino",
+	"build": {
+		"dockerfile": "Dockerfile",
+		"args": {
+			// Update the VARIANT arg to pick a Java version: 11, 15
+			"VARIANT": "11",
```

Comment:
Not sure if it matters but it's 11 here but 15 in the dockerfile.

Reply:
I believe 11 will override the default 15 in the Dockerfile.  Can't wait for 17 to be an option as well...

Reply:
Correct. The 15 in the Dockerfile is the default value (carried over from the devcontainer I copied at the start). This 11 overrides it and is what we want for Trino.